Dockerfile

```
@@ -0,0 +1,8 @@
+FROM debian:latest
+RUN apt update && apt install -y curl libopencv-dev build-essential ffmpeg clang libssl-dev libclang-dev
```

```suggestion
RUN DEBIAN_FRONTEND=noninteractive apt update && apt install -yq --no-install-recommends curl libopencv-dev build-essential ffmpeg clang libssl-dev libclang-dev ca-certificates python3
```

`ca-certificates` is needed else curl errors out with `curl: (77) error setting certificate file: /etc/ssl/certs/ca-certificates.crt`. 

`python3` is needed for obvious reasons (+ rustup just won't let the install finish)

apt would also randomly fail to fetch packages prior to setting `DEBIAN_FRONTEND=noninteractive`, not sure exactly why but it works now -- don't know if this was a one-off thing, somebody else wld need to test

i don't care if `-q` flag is set